A21-service-config-error-handling.md

```diff
@@ -60,6 +60,10 @@ configuration.
 		resolver, it should retry the query after appropriate backoff). In
 		other words,the channel will remain in state TRANSIENT_FAILURE until a
 		valid service config is received.
+5. If the resolver does not support service config resolution OR if no gRPC
```

Comment:
Let's make it clear that the "OR if gRPC config is received but no service config is chosen" part is specific to the DNS resolver.

Reply:
Done!